packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/shared/audio_wrappers.py

```
@@ -0,0 +1,247 @@
+    model_as_dict,
+)
+from opentelemetry.instrumentation.openai.utils import (
+    _with_audio_telemetry_wrapper,
+    dont_throw,
+    is_openai_v1,
+    start_as_current_span_async,
+)
+from opentelemetry.instrumentation.utils import _SUPPRESS_INSTRUMENTATION_KEY
+from opentelemetry.metrics import Counter, Histogram
+from opentelemetry.semconv.attributes.error_attributes import ERROR_TYPE
+from opentelemetry.semconv_ai import (
```

Flake8 (7.3.0)

[error] 'opentelemetry.semconv_ai.SpanAttributes' imported but unused

(F401)